scripts/update_all.py

```diff
@@ -77,6 +77,9 @@
     cmd = 'python ' + scriptsPath + '/workspace_compose.py ' + paramsAll
     if VERBOSE:print(cmd)
     retValue = os.system(cmd)
+    cmd = 'python ' + scriptsPath + '/workspace_addjson.py ' + paramsAll
```

Comment:
Please update this to reflect non-positional arguments.

Reply:
was this comment mend to be for update_app.py?

Reply:
Yes, this command still contains positional arguments..

Reply:
Sorry, my fault. It is ok.